theoplayer/how-to-guides/android/millicast/getting-started.mdx

```diff
@@ -0,0 +1,52 @@
+# Getting started with Millicast on Android
+
+import Intro from '../../shared/millicast/_getting-started-intro.mdx';
+
+<Intro />
+
+## Usage
+
+1. Follow [our Getting Started guide](../../../getting-started/01-sdks/02-android/00-getting-started.mdx)
+   to set up THEOplayer in your Android app.
+2. Add the `integration-millicast` dependency to your module's `build.gradle`.
+3. Add a `MillicastSource` to your player's source.
+
+### Add the `integration-millicast` dependency
+
+Add the Millicast integration along with the [Millicast SDK](https://docs.dolby.io/streaming-apis/docs/android)
+to your module `build.gradle` file, as demonstrated below:
+
+```groovy
+dependencies {
+    // ...
+    implementation 'com.theoplayer.theoplayer-sdk-android:core:+'
+    implementation 'com.theoplayer.theoplayer-sdk-android:integration-millicast:+'
+    implementation "com.millicast:millicast-sdk-android:2.0.0"
+    // ...
+}
+```
+
+### Add a `MillicastSource`
+
+After setting up a `THEOplayerView` in your app's activity, set its source to a `SourceDescription` containing a `MillicastSource`.
+You'll need a Millicast account ID and stream name to create a `Credential` object and identify your Millicast stream:
+
+```kotlin
+import com.millicast.subscribers.Credential
+import com.theoplayer.android.api.millicast.MillicastSource
+
+theoplayerView.player.source = SourceDescription.Builder(
+    MillicastSource(
+        credential = Credential(
+            streamName = "multiview",
+            accountId = "k9Mwad",
+            apiUrl = "https://director.millicast.com/api/director/subscribe"
+        )
+    )
+).build()
+```
+
+## More information
+
+- [API references](pathname:///theoplayer/v8/api-reference/android/com/theoplayer/android/api/millicast/package-summary.html)
```

Comment:
[This link](https://www.theoplayer.com/docs/theoplayer/v8/api-reference/android/com/theoplayer/android/api/millicast/package-summary.html) is currently dead. We had issues with publishing JavaDoc for the Millicast integration during the last release, and we still haven't figured out how to fix that...

If we can't resolve it, we'll have to remove this link for now.

Reply:
Curious 🤔 Let's figure that out, since we're not directing people to this page just yet I'm ok publishing as is while we figure it out and fix it later cause I'd avoid the hassle / risk of forgetting of having to come back and reinstate it but if you prefer that, also good for me.

Reply:
This should be fixed in the next THEOplayer release. Let's wait for that to land before merging this PR.